internal/provider/resource_ko_build.go

```diff
@@ -119,6 +119,13 @@ func resourceBuild() *schema.Resource {
 				Elem:        &schema.Schema{Type: schema.TypeString},
 				ForceNew:    true, // Any time this changes, don't try to update in-place, just create it.
 			},
+			"tags": {
+				Description: "Which tags to use for the produced image instead of the default 'latest' tag",
+				Optional:    true,
```

Comment:
Would be good to explicitly expose the default here

Reply:
Unfortunately I was not able to make it take a default using the native `DefaultFunc` method.

Here's what I tried:
```go
DefaultFunc: func() (interface{}, error) {
	defaultTags := []string{"latest"}
	tagsInterfaces := make([]interface{}, len(defaultTags))
	for i, v := range defaultTags {
		tagsInterfaces[i] = v
	}
	return tagsInterfaces, nil
},
```

Reply:
Fair enough, but unfortunate :-/